.github/workflows/macos-auditbeat.yml

```diff
@@ -13,6 +13,9 @@ on:
 env:
   BEAT_MODULE: 'auditbeat'
 
+permissions:
```

Comment:
@v1v I'm unsure if this needs `actions: write` because of the `v1v/otel-upload-test-artifact-action@v2` action step.

Reply:
If you don't have a quick answer to this I will verify it somehow.

Reply:
Okay just checked. I couldn't get it fully working on my test repo.

but according to https://www.npmjs.com/package/@actions/artifact `actions: read` is sufficient.

Also using `actions: read` on my test repo got rid of the `HttpError: Resource not accessible by integration` error.

Reply:
Oh, I just double checked.. and the `actions: read` permissions are only necessary on private repos.